### PR TITLE
Quest Rework Part 3 + Some Fixes

### PR DESCRIPTION
**Pull request by Kanzaji**
Part 3 of my quest rework started back in the summer of 2022 😅
Introducing:

- [x] Finished Create quests
- [x] Finished Ars Nouveau quests (Ported from E9)
- [x] Challenges!
- [x] Some random fixes
- [x] New Trophies
- [x] Reworked Section of Iron Jetpacks (+ New Enderium Jetpack)
- [x] Formatting test!
obraz
obraz

This PR resolves:
#334
#332
#330
#267
#265 

To Do:
- [x] Fix up IF Infinite Tools related trophies, waiting for dev of Trofes to reply.

### CODE REVIEW

**Comment by Kanzaji** [comment]
Also, I forgot to send a preview of new Trophies ^^"
The file is pretty big, so here is a link for anyone that wants to see it:
https://cdn.discordapp.com/attachments/730169148294037544/1061374487611834369/2023-01-07_21-01-47.mp4
For Contributors, the video is on the #contributor-chat
https://canary.discord.com/channels/195234321836146688/730169148294037544/1061374489700602029

**Comment by Kanzaji** [comment]
Alrighty, got a response from Trofes Dev, and I was able to get info why the trophies with IF sounds didn't want to work ^^"
It seems that client-sided sound effects can not be used for Trophies sounds.
Ready to review / merge 😄

**Comment by Kanzaji** [comment]
So I decided to do a bit more of Formatting, and would need some opinions about it ^^"
I would love to add some colors and decorations to all the quests, but that will take really long time, and I think it is better if I would do so in Enigmatica 9.
obraz
obraz
obraz
obraz
obraz
PS: Chapters are possible to just click, and they will open correct quest lines!

**Comment by NielsPilgaard** [comment]
I agree that large scale improvements would be a better fit in E9.

> PS: Chapters are possible to just click, and they will open correct quest lines!

That is freaking awesome.

**Comment by Kanzaji** [comment]
Alrighty! Gonna work on formatting for E9 quests as well, so now question. Should I revert formatting changes to the Welcome Chapter, or leave it like that for E8?

File: config/ftbquests/quests/chapters/tools.snbt [hunk 1]
```diff
@@ -3079,62 +3101,100 @@
 			y: -16.0d
 			subtitle: "Iron jetpacks have only 1 iron jetpack?!? Ironic."
 			description: [
-				"Second tier of jetpacks are made of Bronze, Silver and wait for it, Iron! (Finally) Of course you need any jetpack from Tier 1 to craft Tier 2."
+				"Second tier of jetpacks introduce &ofinally&r Iron jetpack!"
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3104]: `Second tier of jetpacks introduce finally Iron jetpack!` Is grammatically incorrect. Not sure what you were going for here tbh - `Second tier of jetpacks finally introduce the Iron Jetpack`?

File: kubejs/data/enigmatica/trofers/creative_energy.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "name": {
+        "text": "Infinite Energy Trophy",
+        "color": "#9336a3"
+    },
+    "tooltip": [
+        {"text": "Charge fully Ultimate Induction Cell."}
```
Comment (NielsPilgaard) [inline kubejs/data/enigmatica/trofers/creative_energy.json:7]: `Fully Charged an Ultimate Induction Cell`

File: kubejs/data/enigmatica/trofers/if_backpack.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": {
+        "text": "Infinity Backpack Trophy",
+        "color": "#9d6cb8"
+    },
+    "tooltip": [
+        {"text": "Obtain Infinity Backpack with Artifact tier!"}
```
Comment (NielsPilgaard) [inline kubejs/data/enigmatica/trofers/if_backpack.json:7]: `Obtain an Artifact Tier Infinity Backpack`

**Review by NielsPilgaard — CHANGES_REQUESTED** [review]
I'm sorry for asking this, but... I'd like you to use some sort of grammar checker when writing quests, there are quite a few grammar issues in here.

**Comment by NielsPilgaard** [comment]
> Alrighty! Gonna work on formatting for E9 quests as well, so now question. Should I revert formatting changes to the Welcome Chapter, or leave it like that for E8?

Leave it like that for E8 please, it looks good :)

File: config/ftbquests/quests/chapters/tools.snbt (hunk 1 above)
Comment (Kanzaji) [reply config/ftbquests/quests/chapters/tools.snbt:3104]: Yea... I don't fully like those iron jetpack quests >.> But I honestly think the old ones were worse.
Gonna fix it up 👍

**Comment by Kanzaji** [comment]
That is pretty much what happens when I try to do quest work when I'm tired 😅 Anyway typos fixed

File: config/ftbquests/quests/chapters/tools.snbt [hunk 4]
```diff
@@ -3158,72 +3216,125 @@
 			y: -17.0d
 			subtitle: "No, wait-it's actually too much signal."
 			description: [
-				"Third tier jetpacks are made of Electrum, "
-				"Gold, Invar and Steel. All shocking materials (but not exactly). Again Tier 2 is required to make Tier 3 one."
+				"Third tier is a bit tricky, as it requires some sort of alloy metal! It also has the biggest variety of jetpacks to chosoe from."
 				""
-				"Gold and Invar are again the same, Steel is a bit better in everything, and Electrum is the best one from tier 3!"
+				"Alloys can be created with use of many things. Thermal Induction Smelter, Tinker's Construct Smeltery or Create Heated Mixing! Check JEI and other quest chapters for specifics."
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3221]: `Alloys can be created with use of many things.` -> `Alloys can be created in many ways`

File: config/ftbquests/quests/chapters/tools.snbt [hunk 5]
```diff
@@ -3245,58 +3356,72 @@
...
 			x: -6.5d
 			y: -17.5d
-			subtitle: "RF? This is mysterious? Platinum is mysterious pff"
-			description: [
-				"Fourth tier is made of Diamonds and Platinum. Once again, Tier below is needed to get Tier 4."
-				""
-				"Even tho Platinium is much more expensive than Diamond, it is only a tiny bit better in terms of speeds than it."
-			]
+			subtitle: "RF? This is mysterious? Wasn't RF just FE..."
+			description: ["Fourth tier is the high-end of your upgrade path, giving you a choice of 2 jetpacks from premium materials. Which one you should choose?"]
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3360]: `Which one you should choose?` -> `Which one should you choose?`

`Which one will you choose?` makes a bit more sense though, imo

File: config/ftbquests/quests/chapters/tools.snbt [hunk 6]
```diff
@@ -3319,42 +3444,41 @@
 			x: -5.5d
 			y: -19.5d
 			shape: "gear"
-			subtitle: "How is emerald better than Platinium!?"
-			description: ["The final tier of jetpack is created from Emeralds. Seriously who thought this is.. Fine I will stop. Have fun flying! No, I don't want a ride, you keep your supersonic jetpack to yourself, thanks."]
+			subtitle: "SuperSonic Boom incoming in 3... 2... 1..."
+			description: ["The final tier of jetpack is created from Emeralds. Seriously who thought this is.. Fine I will stop. Have fun flying! No, I don't want a ride, you keep your supersonic jetpack to yourself, Thanks."]
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3448]: `Thanks` should be `thanks`

File: config/ftbquests/quests/chapters/tools.snbt [hunk 7]
```diff
@@ -3158,72 +3216,125 @@
 			y: -17.0d
 			subtitle: "No, wait-it's actually too much signal."
 			description: [
-				"Third tier jetpacks are made of Electrum, "
-				"Gold, Invar and Steel. All shocking materials (but not exactly). Again Tier 2 is required to make Tier 3 one."
+				"Third tier is a bit tricky, as it requires some sort of alloy metal! It also has the biggest variety of jetpacks to chosoe from."
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3219]: `alloy metal` should be either `metal alloy` or just `alloy`

File: config/ftbquests/quests/chapters/tools.snbt [hunk 8]
```diff
@@ -3018,55 +3019,76 @@
 			x: -4.5d
 			y: -15.5d
 			subtitle: "Close enough"
-			description: ["First tier of jetpacks are made from wood, stone and copper. Wood is the basic one, needed for anything better, second is stone and the best is copper!"]
+			description: [
+				"First tier of jetpacks are made from very common materials, what makes them great for early game flying!"
+				""
+				"Wooden Jetpack (Tier 0) is the starting point of your jetpack upgrade path. There is 5 tiers of jetpacks in total, and each tier requires a jetpack from the lower tier for crafting."
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3025]: `Wooden Jetpack (Tier 0) is the starting point of your jetpack upgrade path. There is 5 tiers of jetpacks in total, and each tier requires a jetpack from the lower tier for crafting.` -> `Wooden Jetpack (Tier 0) is the starting point of your jetpack upgrade path. There are 5 tiers of jetpacks in total, and each tier requires a jetpack from the previous tier to craft.`

File: config/ftbquests/quests/chapters/tools.snbt [hunk 9]
```diff
@@ -3018,55 +3019,76 @@
 			x: -4.5d
 			y: -15.5d
 			subtitle: "Close enough"
-			description: ["First tier of jetpacks are made from wood, stone and copper. Wood is the basic one, needed for anything better, second is stone and the best is copper!"]
+			description: [
+				"First tier of jetpacks are made from very common materials, what makes them great for early game flying!"
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/tools.snbt:3023]: `First tier of jetpacks are made from very common materials, what makes them great for early game flying!` -> `First tier of jetpacks are made from very common materials, which makes them great for early game flying!`

File: kubejs/data/enigmatica/trofers/lava.json [hunk 10]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": {
+        "text": "Infinite Lava Trophy",
+        "color": "#ed671a"
+    },
+    "tooltip": [
+        {"text": "Fill entire Supreme Black Hole Tank with Lava."}
```
Comment (NielsPilgaard) [inline kubejs/data/enigmatica/trofers/lava.json:7]: `Fill entire Supreme Black Hole Tank with Lava.` -> `Fill an entire Supreme Black Hole Tank with Lava.`

File: kubejs/data/enigmatica/trofers/tame_all_animals.json [hunk 11]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": {
+        "text": "Zoology Trophy",
+        "color": "#f91d80"
+    },
+    "tooltip": [
+        {"text": "Tame all animals! And here is your special cat for that."}
```
Comment (NielsPilgaard) [inline kubejs/data/enigmatica/trofers/tame_all_animals.json:7]: `And here is your special cat for that.` what?

**Review by NielsPilgaard — CHANGES_REQUESTED** [review]
Was the Jetpack change intentional? 😄

File: kubejs/data/enigmatica/trofers/tame_all_animals.json (hunk 11 above)
Comment (Kanzaji) [reply kubejs/data/enigmatica/trofers/tame_all_animals.json:7]: The trophy is literally a cat on purple carpet 😅
obraz

`And here is your special cat for that.` -> `And here is your own special cat as a reward.` Maybe ^^"? Or I should just remove that bit.

**Comment by Kanzaji** [comment]
> Was the Jetpack change intentional? 😄
Yea ^^" After a bit of thinking it didn't make much sense tbh.

File: kubejs/data/enigmatica/trofers/tame_all_animals.json (hunk 11 above)
Comment (NielsPilgaard) [reply kubejs/data/enigmatica/trofers/tame_all_animals.json]: Just remove that bit imo

Comment (Kanzaji) [reply kubejs/data/enigmatica/trofers/tame_all_animals.json]: Alrighty.

**Review by NielsPilgaard — APPROVED** [review]
Looks good, thanks! Ready for merging?

**Comment by Kanzaji** [comment]
I've read those changes 4 times and can't find anything, so I would say yes ^-^
Would love to get some more formatting work done, but like we decided earlier, I think it is better if I do that for Enigmatica 9.

**Pull request merged**